Packs/Sepio/ReleaseNotes/SeprioPrimeAPI_FromEventId.md

```diff
@@ -0,0 +1,34 @@
+Sepio Prime API Test Playbook with all the integration supported commands
```

Comment:
In this folder supposed to be the release notes file, not the readme. It should be named by the version of the pack (for instance, `1_0_6.md`) and should include a list of changes.
Take an example from the other files in this folder.